roles/inspec/defaults/main.yml

```diff
@@ -14,6 +14,13 @@ inspec:
       name: inspec-stig-rhel7
       version: master
       url: https://github.com/inspec-stigs/inspec-stig-rhel7/archive/master.tar.gz
+    ursula-inspec:
+      name: ursula-inspec
+      version: master
+      url: https://github.com/blueboxgroup/ursula-inspec/archive/master.tar.gz
+      sub_profiles:
+        - name: inspec-mysql
+          path: /ursula-inspec/mysql
```

Comment:
don't have a prefix of `/` here as its a relative path and the `/` might be confusing.